finetune/model.py

```diff
@@ -320,7 +295,7 @@ def optimizer(lr):
                 increment_global_step=True,
                 summaries=summaries,
                 colocate_gradients_with_ops=True,
-                variables=params.trained_variables,
+#                variables=trained_variables,
```

Comment:
Yea seems like we can cut this out entirely.  Default seems to be train all trainable params which is what we did explicitly before.